datasets/surftemp-sst.yaml

```diff
@@ -0,0 +1,34 @@
+Name: "Sea Surface Temperature Daily Analysis: European Space Agency Climate Change Initiative product version 2.1"
+Description: "Global daily-mean sea surface temperatures, presented on a 0.05° latitude-longitude grid, with gaps between available daily observations filled by statistical means, spanning late 1981 to recent time.  Suitable for large-scale oceanographic meteorological and climatological applications, such as evaluating or constraining environmental models or case-studies of marine heat wave events. Includes temperature uncertainty information and auxiliary information about land-sea fraction and sea-ice coverage. For reference and citation see: www.nature.com/articles/s41597-019-0236-x."
+Documentation: https://surftemp.net/
+Contact: c.j.merchant@reading.ac.uk
+ManagedBy: n.f.mccarroll@reading.ac.uk
```

Comment:
I suggest putting the university or research group here and linking to the relevant website